### PR TITLE
Properly working brute-force algorithm

### PR DESCRIPTION
**Pull request by lukasz-smogorzewski**

### CODE REVIEW

File: src/data_structure/board.java [hunk 1]
```diff
@@ -19,7 +20,7 @@ public board(byte width, byte height, field2D[][] startBoard) {
         this.width = width;
         this.height = height;
 
-        gameBoard = new field2D[width][height];
+        gameBoard = new field2D[width][height];//nie wszedzie zachowalem konwencje [width][height]
```
Comment (zetraxus) [inline src/data_structure/board.java:23]: Could you explain?

File: src/data_structure/board.java [hunk 2]
```diff
@@ -135,7 +359,8 @@ private void updatePossibilitiesByFilledFields(int x, int y) {
         }
     }
 
-    private void updatePossibilities(int x, int y, int value) {
+    /*private void updatePossibilities(int x, int y, int value) {
+        if(1>0)return;
```
Comment (zetraxus) [inline src/data_structure/board.java:363]: :)

File: src/solver/solver.java [hunk 3]
```diff
@@ -17,57 +19,57 @@ public int compare(gameState o1, gameState o2) {
...
-        board.setField(6, 4, 2);
-        board.setField(4, 4, 1);
-        board.setField(3, 4, 3);
-        board.setField(3, 3, 6);
-        board.setField(1, 2, 9);
-        board.setField(2, 2, 8);
-        board.setField(1, 3, 7);
-        board.setField(2, 3, 3);
-        board.setField(4, 3, 9);
+        tu:
```
Comment (zetraxus) [inline src/solver/solver.java:56]: Could you explain that?

File: src/solver/solver.java [hunk 4]
```diff
@@ -17,57 +19,57 @@ public int compare(gameState o1, gameState o2) {
...
+        while (queue.isEmpty() == false) {
+            if ((analyzed = queue.remove()).isSolved()) {
+                System.out.println("" + analyzed.isSolved() + " " + analyzed.getBoardShortcut());
+                System.out.println(new board(template).generate(analyzed.getBoardShortcut()));
+                break tu;
```
Comment (zetraxus) [inline src/solver/solver.java:61]: Why not 'break;'?

File: src/data_structure/board.java (hunk 1 above)
Comment (lukasz-smogorzewski) [reply src/data_structure/board.java:23]: It is possible that somewhere is phrase "gameBoard[height][width];". So i leave this comment to remember about checking correctness before merging branch into master.

File: src/solver/solver.java (hunk 3 above)
Comment (lukasz-smogorzewski) [reply src/solver/solver.java:56]: It should be deleted. In previous version was a nested loop.

File: src/data_structure/board.java (hunk 2 above)
Comment (lukasz-smogorzewski) [reply src/data_structure/board.java:363]: Now, this commented function is for sure unnecessary and it can be deleted.

**Pull request merged**